README.md

```diff
@@ -2,40 +2,78 @@
 
 ![a list of builds in the Travis CI plugin for Backstage](./docs/travis-ci-plugin-1642x1027.png)
 
-[https://travis-ci.com/](https://travis-ci.com/)
+[https://roadie.io/backstage/plugins/travis-ci](https://roadie.io/backstage/plugins/travis-ci)
 
-## Setup
+## Features
+
+- List Travis CI Builds
+- Retrigger builds
 
-1. If you have standalone app (you didn't clone this repo), then do
+## How to enable Travis-ci plugin in Backstage
+
+Travis-ci plugin is a built-in part of vanilla backstage project. To start using it for your component, you have to:
+
+1. add annotation to the yaml config file of a component:
+
+```yml
+travis-ci.com/repo-slug: <owner-name>/<project-name>
+```
+
+2. add your developer api key (from https://travis-ci.com/account/preferences) to the environmental variables for your backstage backend server:
+   `TRAVISCI_AUTH_TOKEN="token <your-api-key>"`
+
+## How to add Travis-ci project dependency to Backstage app
+
+If you have your own version of backstage without this plugin, here it's how to add it:
```

Comment:
> your own version of backstage

This should be "your own backstage application".

The terminology is that users of Backstage will use the @backstage/create-app CLI to create their own "Backstage application". If they just want to try out Backstage, then they may use the "Backstage sample app" which comes inside the Backstage repo.

You totally have the idea right, just slightly different terminology than the Backstage team seem to use.

Reply:
I think we need to make a similar distinction on the marketing site?